packages/gatsby-theme-newrelic/README.md

```diff
@@ -230,6 +231,10 @@ module.exports = {
 };
 ```
 
+#### `gdprTracking`
```

Comment:
A couple things here:

1) This configuration is not used in this PR, so having this listed in the README make confuse others. That being said, if we could only make the `gaTrackingId` configurable, then it would be great to list that here.
2) Can you make sure this gets added to the TOC above?